include/inviwo/core/common/inviwomodule.h

```diff
@@ -255,6 +254,8 @@ class IVW_CORE_API InviwoModule {
         std::unique_ptr<BaseRepresentationConverterFactory> converterFactory);
 
     void registerSettings(std::unique_ptr<Settings> settings);
```

Comment:
We should document what happens with ownership for these three

Reply:
the type says it all

Reply:
agree with @petersteneteg here, in particular when considering lines 313-314

Reply:
At least state if the unregisterSettings *must* be called when using registerSettings(Settings*)